### PR TITLE
Feature/home page updates

### PR DESCRIPTION
**Pull request by Darthmaul**
Updating the homepage layout a bit, per this card: https://trello.com/c/JHnbMbQz/105-home-page-updates

Also, making loadouts on /browse open up in a new tab per this card: https://trello.com/c/H41FYEFF/95-allow-loadouts-in-browse-to-be-opened-in-new-tabs

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
...
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
+                ['character_id', $characterId],
+                ['created_at', '>', Carbon::now()->subDays(14)], // Loadouts created in the past 2 weeks
+            ])
+                ->withCount('votes')
+                ->with('character', 'creator', 'mods.gun')
+                ->orderBy('created_at', 'desc')
```
Comment (chaseconey) [inline app/Http/Controllers/DashboardController.php:82]: You don't have to change it, more of a TIL maybe for ya: you can use the `latest()` function for this as well.

File: resources/views/dashboard/index.blade.php [hunk 2]
```diff
@@ -14,9 +14,34 @@
...
+            @foreach($recentTopLoadouts as $characterLoadouts)
+            <div class="loadoutCards w-full lg:w-1/2 px-1">
+                @foreach($characterLoadouts as $loadout)
+                <x-dashboard-loadout :loadout="$loadout"/>
+                @endforeach
+            </div>
+            @endforeach
+        </div>
+    </div>
+    <div class="featuredLoadoutsContainer">
```
Comment (chaseconey) [inline resources/views/dashboard/index.blade.php:28]: Do we want to extract this whole block into a component? Might clean this up nicely.... not a must do.

File: app/Http/Controllers/DashboardController.php [hunk 3]
```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
 
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
```
Comment (chaseconey) [inline app/Http/Controllers/DashboardController.php:49]: I wonder if we change it up a bit. We could change this to be character agnostic. Like, let's just put the top 10 loadouts up from any class for last 2 weeks. That will kind of break up the colors and kind of ... predictable nature of the dashboard without re-working it majorly.

File: app/Http/Controllers/DashboardController.php [hunk 4]
```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
...
+                ->orderBy('votes_count', 'desc')
+                ->take(3)
+                ->get();
+            $loadouts->push($characterLoadouts);
+        }
+
+        return $loadouts;
+    }
+
+    private function getLatestLoadouts()
```
Comment (chaseconey) [inline app/Http/Controllers/DashboardController.php:70]: Same for this, we should make it class agnostic imo. The only one we leave is maybe the top per character that we had before and that goes at the bottom.

File: app/Http/Controllers/DashboardController.php [hunk 5]
```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
...
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
+    {
+        $loadouts = collect();
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
```
Comment (chaseconey) [inline app/Http/Controllers/DashboardController.php:55]: We can definitely extract this and we can just pass a parameter for order and last n weeks or days. Can add a TODO if you aren't feeling it and I can knock it out.

**Review by chaseconey — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (Darthmaul) [reply app/Http/Controllers/DashboardController.php:82]: Did this locally, will push up shortly. Good note thanks 👍

File: resources/views/dashboard/index.blade.php (hunk 2 above)
Comment (Darthmaul) [reply resources/views/dashboard/index.blade.php:28]: Knocked this out locally as well

**Review by chaseconey — APPROVED** [review]

**Pull request merged**